apps/web/screens/Login/Login.tsx

```diff
@@ -75,7 +76,12 @@ const LoginPage: Screen<LoginProps> = ({ namespace }) => {
                 )}
               </Text>
               <div>
-                <a tabIndex={-1} className={styles.btnLink} href="/minarsidur">
+                <a
+                  tabIndex={-1}
+                  className={styles.btnLink}
+                  onClick={() => webLoginButtonSelect('New')}
```

Comment:
ok, im guessing command clicking or opening in a different tab don't matter in this case?

Reply:
It does not. 👍